src/helpers.rs

```diff
@@ -19,7 +19,7 @@ lazy_static! {
             locale
         }
     };
-    static ref EN_US_LOCALE: Vec<wchar_t> = { OsStr::new("en-us").encode_wide().collect() };
+    static ref EN_US_LOCALE: Vec<wchar_t> = { OsStr::new("en-us\0").encode_wide().collect() };
```

Comment:
I think this might be able to reuse the `ToWide` trait.

```suggestion
    static ref EN_US_LOCALE: Vec<wchar_t> = "en-us".to_wide_null();
```